### PR TITLE
member added uttam

### PR DESCRIPTION
**Pull request by kachwalUttamsharma**

### CODE REVIEW

File: members/uttam/data.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "id": "kachwaluttamsharma",
```
Comment (swarajpure) [inline members/uttam/data.json:2]: id should be `uttam` 
This is RDS id, not github id

**Review by swarajpure — CHANGES_REQUESTED** [review]
Thanks for this PR!

Please crop the photo to your face according to the rules:
> Images should be a square
Size of the image should be greater than 500px X 500px
The face should be visible, centered and identifiable
Face should cover at least 60% of the image

Please check the other profiles to get an idea: https://members.realdevsquad.com/

File: ids/mapping.json [hunk 2]
```diff
@@ -30,5 +30,6 @@
   "swaraj": "swarajpure",
   "sweta": "karnsweta92",
   "taksh": "takshch",
-  "tejas": "tejasberi"
+  "tejas": "tejasberi",
+  "uttam": "Uttam"
```
Comment (swarajpure) [inline ids/mapping.json:34]: The key is correct, the value should be your github id

**Review by swarajpure — CHANGES_REQUESTED** [review]
One correction in the json file
Also please crop the photo while keeping your face in the center. On members page the photo gets cropped in a circle. Some part of your head might get cropped if we merge this PR as it is right now 😅

File: ids/mapping.json (hunk 2 above)
Comment (kachwalUttamsharma) [reply ids/mapping.json:34]: kachwalUttamsharma
This is my github id. updating it.

**Review by swarajpure — APPROVED** [review]
Thank you! Welcome to the members page! 🎉

**Pull request merged**